### DISCUSSION TITLE
Where can I find the v0.22 libs?

### DISCUSSION THREAD

**Discussion by Swede001Labs** [mapsforge/mapsforge#1565] [Q&A] [answered]
Hi, I'm trying to update my build.gradle for an Android project to reference the new 0.22 version but the libs cannot be found. Maybe 0.22 isn't released yet? I use maven as the repository, should I use something else? Here is an example of my issues:

**build.gradle**
_implementation 'org.mapsforge:mapsforge-core:0.22.0'_

**output**
_Could not find org.mapsforge:mapsforge-core:0.22.0._

**Comment by devemux86** [comment, marked as answer]
You need to use the JitPack repository and update the declarations.
See the detailed instructions in the [integration guide](https://github.com/mapsforge/mapsforge/blob/master/docs/Integration.md).

**Reply by Swede001Labs** [reply]
Thanks! After some struggling in Android Studio I got it to work!

**Reply by schwarzstoff** [reply]
How exactly did you get it to work?
When I try to use version 0.22.0 via JitPack, nothing is found — for example:
https://jitpack.io/com/github/mapsforge/mapsforge/mapsforge-themes/0.22.0/mapsforge-themes-0.22.0.jar
returns 404, while higher versions like 0.23.0 are available and work.

Also, using:
implementation 'org.mapsforge:mapsforge-core:0.22.0'
fails to resolve.

I do have maven { url "https://jitpack.io" } set in my repositories block.
Any idea what I might be missing?

**Reply by devemux86** [reply]
@schwarzstoff 

You need to follow the instructions in the [integration guide](https://github.com/mapsforge/mapsforge/blob/master/docs/Integration.md) and use the correct package for each repository.

The JitPack instructions also exist at the [JitPack website](https://jitpack.io/#mapsforge/mapsforge).

**Reply by schwarzstoff** [reply]
Thanks – I followed the integration guide and JitPack instructions exactly as described. Everything works fine with versions above 0.22.0, but unfortunately, I specifically need to support version 0.22.0.

**Reply by schwarzstoff** [reply]
I was building the 0.22.0 local now, that's the only solution, that worked.
